### PR TITLE
Fixes call to get current utc time.

### PR DESCRIPTION
**Pull request by marcovc**
Fixes a mistake in querying utc time (coincidently when I tested this, the current code was returning the right thing :) . Sorry about that!

### CODE REVIEW

File: solver/src/http_solver.rs [hunk 1]
```diff
@@ -320,7 +320,7 @@ impl HttpSolver {
     }
 
     pub fn generate_instance_name(network_id: &str, chain_id: u64) -> String {
-        let now = chrono::offset::Utc::now();
+        let now = chrono::Utc::now();
```
Comment (vkgnosis) [inline solver/src/http_solver.rs:323]: Does this really fix anything? From looking at the chrono code I believe this two calls are the same thing.

Comment (marcovc) [reply solver/src/http_solver.rs:323]: I'm slightly confused about the time in rust. I've added a log message (in other PR) to print the name of the instance I'm creating with the results from this function. I just saw this:

```
2021-06-09T08:43:13.508Z  INFO solver::http_solver: http solver instance name is 2021-06-09_08:26:22.218115801_UTC_Ethereum_1
```

Note the discrepancy between the log time and the instance name. This is the first time I see a discrepancy of minutes (is even more puzzling) but I was seeing discrepancy of hours before. While hovering over this function in vscode, I got the impression that it was computing the difference between local time and utc time:

```
chrono
mod offset

The time zone, which calculates offsets from the local time to UTC.
```

But now I see the problem is something else.

Comment (vkgnosis) [reply solver/src/http_solver.rs:323]: I see what the problem is. `instance_name` is set in `struct SolverConfig`. This is only created once and never updated so the instance name is always this exact string and it is set to the current time when the Solver struct is instantiated. What we want is to always use the current time when solving.

Comment (vkgnosis) [reply solver/src/http_solver.rs:323]: I would in Http::solver send call generate_instance_name and then add this to the url query part in line 274. You can see how `Config` adds stuff to the query. Config was meant as static global configuration.

Comment (marcovc) [reply solver/src/http_solver.rs:323]: oh... of course, now everything makes sense. Will close this and open another PR with the fix. Thanks!

**Comment by marcovc** [comment]
No need to fix anything.

**Pull request closed without merging**